<ui version="4.0" >
 <class>VocableEdit</class>
 <widget class="QDialog" name="VocableEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VocableEditor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="4" column="0" >
        <widget class="QLabel" name="lastQueryLabelLeft" >
         <property name="text" >
          <string>Last Query:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="VocableEditorTextEdit" name="foreignTextEdit" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="boxLabel" >
         <property name="text" >
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="boxLabelLeft" >
         <property name="text" >
          <string>Box:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="foreignLabel" >
         <property name="text" >
          <string>&amp;Foreign:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>foreignTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="nativeLabel" >
         <property name="text" >
          <string>&amp;Native:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>nativeTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="VocableEditorTextEdit" name="nativeTextEdit" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="lastQueryLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Lession:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="lessionComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="translateGroupBox" >
      <property name="title" >
       <string>lookup vocable at leo.de</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="translateTextBrowser" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VocableEditorTextEdit</class>
   <extends>QTextEdit</extends>
   <header>VocableEditorTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nativeTextEdit</tabstop>
  <tabstop>foreignTextEdit</tabstop>
  <tabstop>lessionComboBox</tabstop>
  <tabstop>translateTextBrowser</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>VocableEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>543</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>VocableEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>624</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
